docs/apache-airflow-providers-openlineage/index.rst

```diff
@@ -93,10 +93,16 @@ for the minimum Airflow version supported) via
 Requirements
 ------------
 
-==================  ==================
-PIP package         Version required
-==================  ==================
-``apache-airflow``  ``>=2.6.0``
-==================  ==================
+The minimum Apache Airflow version supported by this provider package is ``2.6.0``.
```

Comment:
cc: @eladkal example documentation generated for non-preinstalled package. Also shows that packages (like openlineage) can have different min-airflow-version set (exceptions are dict in prepare_providers_packages.py). Open-lineage has 2.6.0 as min airflow version and all nicely works here.